### PR TITLE
updates punctuation for translations

### PR DESCRIPTION
**Pull request by jessleenyc**
#### What's this PR do?
updates po & pot files. added strings to POEditor to account for punctuation, translated and exported the po file to be uploaded. 

#### How should this be reviewed?
...

#### Any background context you want to provide?
...

#### Relevant tickets
Fixes #

#### Checklist
- [ ] Documentation added for new features/changed endpoints.
- [ ] Tested on staging.
- [ ] Pinged a PM if this is a larger PR that would benefit from some additional testing love
- [ ] Post a message in #phoenix if this includes something that causes a rebuild!

### CODE REVIEW

**Comment by jessleenyc** [comment]
@weerd just realized that I have no idea where all the 'month of the year' stuff is coming from?

File: translations/pos/Phoenix_Portuguese_BR.po [hunk 1]
```diff
@@ -2006,3 +2006,15 @@ msgstr "Junte-se a outros e faça a diferença"
...
 msgstr "O que é DoSomething.org? "
 
+#: Brand Slogan
+msgid "Let's Do This!mgstr \""
+msgstr ""
+
+#: Brand Slogan
+msgid "Let's Do This!"
+msgstr "Vamos em frente!"
+
```
Comment (weerd) [inline translations/pos/Phoenix_Portuguese_BR.po:2012]: Why are there two of these with one as `Let's Do This!mgstr \"`??

File: translations/pos/Phoenix_Spanish_MX.po [hunk 2]
```diff
@@ -1998,3 +1998,15 @@ msgstr "Únete a los demás para hacer una diferencia"
...
 msgstr ""
 
+#: Brand Slogan
+msgid "Let's Do This!mgstr \""
+msgstr ""
+
+#: Brand Slogan
+msgid "Let's Do This!"
+msgstr "¡Hagámoslo!"
+
```
Comment (weerd) [inline translations/pos/Phoenix_Spanish_MX.po:2004]: Same as the Portuguese one... :\

**Review by weerd — CHANGES_REQUESTED** [review]
Comment on repeated key/value items. Maybe check in POEditor to see if two exist and likely delete the one w/ the `msgstr \"`??

**Comment by jessleenyc** [comment]
@weerd re-uploaded

**Review by weerd — APPROVED** [review]
👍

**Pull request merged**